Patch-Source: https://github.com/kpcyrd/rebuilderd/commit/be3e43e6c88fe50a4877402e70eb22b37020f6f4.patch
--
From be3e43e6c88fe50a4877402e70eb22b37020f6f4 Mon Sep 17 00:00:00 2001
From: Vekhir <134215107+Vekhir@users.noreply.github.com>
Date: Wed, 11 Oct 2023 16:18:09 +0200
Subject: [PATCH] Match `application/x-bzip2`

shared-mime-info 2.3 yields application/x-bzip2 instead of application/x-bzip
---
 tools/src/decompress.rs | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tools/src/decompress.rs b/tools/src/decompress.rs
index 8f52083..64a3022 100644
--- a/tools/src/decompress.rs
+++ b/tools/src/decompress.rs
@@ -24,6 +24,7 @@ pub fn detect_compression(bytes: &[u8]) -> CompressedWith {
     match mime {
         "application/gzip" => CompressedWith::Gzip,
         "application/x-bzip" => CompressedWith::Bzip2,
+        "application/x-bzip2" => CompressedWith::Bzip2,
         "application/x-xz" => CompressedWith::Xz,
         "application/zstd" => CompressedWith::Zstd,
         _ => CompressedWith::Unknown,
